docs/cli-usage.md

```diff
@@ -150,6 +150,46 @@ Identifier styles (ex: `FooBar`, `fooBar`, `foo_bar`, `FOO_BAR`, `m_fooBar`)
 | `--ident-objc-type-param ...` |
 | `--ident-objc-local ...` |
 | `--ident-objc-file ...` |
+|   |
```

Comment:
You are talking about default values below. Could you document them here and in the `--help` output, so I can see at a glance what the default is on each type? This would be really helpful!

Why did you add the new blank table row at the end?

Reply:
this is possible, but I need to refactor_read this from the source to get the info out
its actually not difficult, but I might take some time

Reply:
I'm fine with opening a new issue for this if it requires some more effort.

Reply:
```suggestion

```

Reply:
I created #25 for the documentation of the default values.

Reply:
ah, yes, this empty line,  I added it there to create an additional line at the end, but my VS code preview looks of course different than the mkdoc one